src/ui/components/LedgerViewer/LedgerViewer.js

```diff
@@ -220,108 +215,3 @@ const GrainReceiptTable = memo(
 );
 
 GrainReceiptTable.displayName = "GrainReceiptTable";
-
-const LedgerEventRow = React.memo(
```

Comment:
Was this tested on a production-sized ledger, or are you planning to memoize Ledger Entries elsewhere in your refactor? I'm pretty sure missing this will have significant impacts when re-rendering larger ledgers when opening and closing the distribution modal.

Reply:
yes, I am addressing this now - I replied to my own review request to say standby while I address this